xbmc/GUIInfoManager.cpp

```diff
@@ -5832,6 +5884,18 @@ bool CGUIInfoManager::ConditionsChangedValues(const std::map<INFO::InfoPtr, bool
   return false;
 }
 
+bool CGUIInfoManager::GetShowVideoOSD() const
+{
+  CGUIWindow *pOSD = g_windowManager.GetWindow(WINDOW_DIALOG_VIDEO_OSD);
+  return pOSD && pOSD->IsDialogRunning();
+}
+
+bool CGUIInfoManager::GetShowMusicOSD() const
```

Comment:
@xhaggi I'm calling this methods from outside <code>CGUIInfoManager</code>, for example here https://github.com/xbmc/xbmc/pull/7379/files#diff-14ed9651a2eb656ff75aea52d0912a00R4112 and wanted to encapsulate the implemantion. I consider <code>CDVDPlayer</code> dealing *directly* with gui window code no good. Thus, I would keep the methods but simplify the implemention. Okay?